packages/core/src/app/common/utility/should-filter-apple-pay.ts

```diff
@@ -0,0 +1,5 @@
+import { isApplePayWindow } from './';
+
+export default function shouldFilterApplePay(methodId: string, isBrowserSupport: boolean): boolean {
+    return Boolean(methodId === 'applepay' && !isApplePayWindow(window) && !isBrowserSupport);
```

Comment:
This already returns a Boolean so wrapping in boolean is unnecessary. Can be cleaner if we can do 

```
if (methodId !== 'applepay') {
        return false;
    }
    
 if (isApplePayWindow(window) && isBrowserSupported) {
     return false;
 }
 
 return true;
```